keras_cv/readme_quickstart_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+import keras_cv
+
+"""
+ATTENTION!!!
+
+This test exists to make sure we don't accidentally break our quickstart pages.
+
+If you change this test you are responsible for creating a PR to update:
+- https://github.com/keras-team/keras-io/blob/master/templates/keras_cv/index.md
+- https://github.com/keras-team/keras-cv/blob/master/README.md#quickstart
+
+Thank you!
+"""
+
+
+@pytest.mark.skipif(
```

Comment:
IIRC we had decided to stop with this strategy and instead use --ignore flags in our CI testing flows to skip tests we don't want to run.

Reply:
it still exists several places that involve fitting models.  Lets fix it all in a batch - but for now I think this really only belongs in GCBRun.  

The tracing is pretty expensive on people's local CPUs and could be a huge painpoint for local devs.

Reply:
lets resolve this and merge, this PR has been shelved for awhile and adds value